pyramid_okta/security/authorization.py

```diff
@@ -0,0 +1,5 @@
+__author__ = 'psuresh'
+from pyramid.authorization import ACLAuthorizationPolicy
+
+# Use the default ACL policy and define permissions as ACL's`
+OktaAuthorizationPolicy = ACLAuthorizationPolicy
```

Comment:
no need for any custom policy at this point. The default ACL policy will check against the ACL's we assign. We can use OktaAuthenticatiojnPolicy to get the principals from Okta and default ACL comparison to check if principal has permission requested by view.